CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## Version 0.0.6
```

Comment:
These correspond to npm package versions, so let's put all of these changes under 0.0.5. I'm planning on publishing 0.0.4 today.